gen/plugins/com.salesforce/com.salesforce.plugin.smartstore.js

```diff
@@ -92,6 +91,7 @@ var StoreCursor = function () {
 
 // ====== Logging support ======
 var logLevel;
+var isGlobalStore;
```

Comment:
logLevel (above) and storeConsole (below) are both for logging support (the previous comment) - so maybe isGlobaStore should be declared before the Logging support comment?